Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -1495,7 +1495,7 @@ BootPickerEntriesSet (
         }
 
         break;
-      case OC_BOOT_EXTERNAL_SYSTEM:
+      case OC_BOOT_UNMANAGED:
```

Comment:
Note that OC_BOOT_EXTERNAL_TOOL and OC_BOOT_SYSTEM are present a few lines above, if you expand upwards. (IMO while the previous name OC_BOOT_EXTERNAL_SYSTEM was similar to both these, the meaning is not similar.)

Reply:
Yes, that feels ok to me.